resources/views/dashboard/training/new_ticket.blade.php

```diff
@@ -111,13 +111,13 @@
             <div class="col-sm-6">
                 <div class="form-group">
                     <label for="comments" class="form-label">Comments (Visible to Controller and other Trainers)</label>
-                    {{ html()->textarea('comments', null)->class(['form-control']) }}
+                    <textarea name="comments" class="form-control text-editor" id="comments"></textarea>
```

Comment:
should have the spatie-styled form markup here... was there a reason that you reverted this to html?